testsuite/do_tests.sh.in

```diff
@@ -847,7 +856,13 @@ echo
 
 if test ${TEST_FAILED} -gt 0 ; then
     echo "***"
-    echo "*** There were errors detected during the run of the NEST test suite!"
+    echo "*** The following tests failed while running the NEST Testsuite:"
+    echo "***"
+    for ft in ${FAILED_TESTS}
+    do
+      echo "***     ${ft}"
+    done
+    echo "***"
```

Comment:
This line is being printed twice if there were no failing tests. However, I'm not sure if this justifies an additional `if`... If you go for it, I'd like to see its body to also include the `for` loop.